course_content/notebooks/matplotlib_intro.ipynb

```diff
@@ -27,49 +27,36 @@
      ],
      "language": "python",
      "metadata": {},
-     "outputs": [],
-     "prompt_number": 1
+     "outputs": []
     },
     {
      "cell_type": "markdown",
      "metadata": {},
      "source": [
       "## The matplotlib figure\n",
       "\n",
-      "At the heart of **every** matplotlib plot is the \"Figure\" object. The \"Figure\" object is the top level concept that can be drawn to one of the many output formats, or simply just to screen. Any object that can be drawn in this way is known as an \"Artist\" in matplotlib.\n",
-      "\n",
-      "Let's create our first artist using pyplot, and then show it:"
+      "At the heart of every matplotlib plot is the \"Figure\" object. The \"Figure\" object is the top level concept that can be drawn to one of the many output formats, or simply just to screen.\n",
+      "Lets create our first Figure object using pyplot, and then show it:"
      ]
     },
     {
      "cell_type": "code",
-     "collapsed": true,
+     "collapsed": false,
      "input": [
       "fig = plt.figure()\n",
       "plt.show()"
      ],
      "language": "python",
      "metadata": {},
-     "outputs": [
-      {
-       "metadata": {},
-       "output_type": "display_data",
-       "text": [
-        "<matplotlib.figure.Figure at 0x43c2550>"
-       ]
-      }
-     ],
-     "prompt_number": 2
+     "outputs": []
     },
     {
      "cell_type": "markdown",
      "metadata": {},
      "source": [
-      "On its own, drawing the figure artist is uninteresting and will result in an empty piece of paper (that's why we didn't see anything above).\n",
-      "\n",
-      "By far the most useful artist in matplotlib is the \"Axes\" artist. The Axes artist represents the \"data space\" of a typical plot. A rectangular axes (the most common axes, but not the only axes, e.g. polar plots) will have two (confusingly named) Axis Artists with tick labels and tick marks.\n",
```

Comment:
"The Axes artist represents the \"data space\" of a typical plot"

Can we keep this line, replacing "artist" with "object"?